java/src/jmri/jmrix/loconet/LnOpsModeProgrammer.java

```diff
@@ -441,7 +477,6 @@ public void confirmCV(String CV, int val, ProgListener pL) throws ProgrammerExce
      */
     @Override
     public void message(LocoNetMessage m) {
-        log.debug("LocoNet message received: {}", m);
```

Comment:
Is this line intended to be removed? One option is to change `log.debug()` to `log.trace()` to limit the amount of output.
https://www.jmri.org/help/en/html/doc/Technical/Logging.shtml